### ISSUE TITLE
how to enable assertion with wisdom ?

### ISSUE THREAD

**Issue by hboumedane** [wisdom-framework/wisdom#423]

**Comment by cescoffier** [comment]
When launched in `prod`, you can configure it using:
`JVM_ARGS = -ea`

**Comment by hboumedane** [comment]
I'm in debug mode , i tried to use this option with wisdom in watch mode: 
```java
-Ddebug=5005 -Dea
```
```maven
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.16</version>
                <configuration>
                    <enableAssertions>true</enableAssertions>
                </configuration>
            </plugin>

```

**Comment by cescoffier** [comment]
Surefire enables the assertions by default. As your test runs in the Surefire JVM, it should work out of the box.

**Comment by hboumedane** [comment]
not working !!!

**Comment by cescoffier** [comment]
Oh, I miss the point where you are in watch mode, then yes, we need to pass the `ea` parameter to the JVM.

**Comment by cescoffier** [comment]
So the best way would be to support a `jvmArgs` parameter to indicate the JVM parameters.

**Comment by cescoffier** [comment]
Fixed with #424 

Just add the following argument:

`mvn clean wisdom:run -Ddebug=5005 -DjvmArgs="-ea"`
